src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
-body {
+$cardWidth: 198px;
+$borderStyle: 1px solid #f3f3f3;
+$linkBlueColor: #00acdc;
+
+@mixin reset-default-styles {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+@mixin font {
+  font-family: "Roboto", "Arial", sans-serif;
+  font-style: normal;
+}
+
+body {
+  @include reset-default-styles();
+  @include font();
+}
+
+.card {
+  border: $borderStyle;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $cardWidth;
+
+  &__img {
+    margin: 32px 19px 40px;
```

Comment:
you give a horizontal margins for every element inside card. maybe better to give padding's for the card instead, and here just margin bottom or top?